app/views/shared/_alternative_map.html.erb

```diff
@@ -0,0 +1,3 @@
+<p class="govuk-body">
+  If you need this information in an alternative format, contact your case officer on <%= @local_authority["feedback_email"] %><%= @local_authority["telephone_number"].present? ? " or by telephone at #{@local_authority["telephone_number"]}." : "." %>
```

Comment:
What happens here if the LA hasn't set their contact details?

Reply:
I think the email is mandatory, and the phone number not?

Reply:
But I should check

Reply:
Somehow I managed to overlook that there's a ternary here checking for that precise situation. 🤦🏻  ignore me.

Reply:
No i've checked and email isn't mandatory either! Will fix 😄